dev/release/verify-release-candidate.sh

```diff
@@ -468,6 +469,16 @@ install_conda() {
   conda activate base
 }
 
+install_maven() {
+  show_info "Installing Maven version ${MAVEN_VERSION}..."
+  APACHE_MIRROR="http://www.apache.org/dyn/closer.cgi?action=download&filename="
```

Comment:
Could you use `https` and `.lua`?

See also: https://infra.apache.org/release-download-pages.html#download-page

```suggestion
  APACHE_MIRROR="https://www.apache.org/dyn/closer.lua?action=download&filename="
```